docs/skipped-urls.txt

```diff
@@ -9,6 +9,7 @@ https://twitter.com/intent/follow*
 https://twitter.com/intent/tweet*
 https://twitter.com/fluidframework
 https://c.s-microsoft.com/static/fonts/segoe-ui/west-european/*
+https://github.com/microsoft/FluidFramework/issues/*
```

Comment:
Probably worth excluding the aka.ms links too since those are blocked by robots.txt.